shim.js

```diff
@@ -5809,22 +5809,22 @@ dnode.prototype = {};
 function dnode (cons, opts) {
```

Comment:
The only changes that have been made should be the addition of :

onPageCreated: function(cb){
        page.onPageCreated = function(new_page) {
          var new_page = pageWrap(new_page);
          return cb(new_page);
        };
      },

to shim.js.

I pulled from the latest repo prior to making the commit.  I have not made a change to the shim.coffee as of yet, will make the change and commit momentarily.



Thanks,
Paul Disney | Software Engineer
Steelgate Intelligence Systems
222 Broadway New York, NY 10038
Tel: +1 443 203 8005
Skype: pdisney1
www.steelgateis.com<http://www.steelgateis.com>​

On Sep 16, 2015, at 11:46 AM, Amir Raminfar <notifications@github.com<mailto:notifications@github.com>> wrote:


In shim.js<https://github.com/sgentle/phantomjs-node/pull/310#discussion_r39646898>:

> @@ -5809,22 +5809,22 @@ dnode.prototype = {};
>  function dnode (cons, opts) {


Where is the change to shim.coffee

—
Reply to this email directly or view it on GitHub<https://github.com/sgentle/phantomjs-node/pull/310/files#r39646898>.